src/helpers/index.js

```diff
@@ -0,0 +1,10 @@
+module.exports={
```

Comment:
use `ES6 syntax`

Reply:
babel was not configured before but now every thing works fine